plugins/europeana/search.js

```diff
@@ -142,6 +142,27 @@ function facetsFromApiResponse(response) {
   return facets;
 }
 
+/**
+ * Page to request from API based on URL query parameter
+ * If parameter is not present, returns default of page 1.
+ * If parameter is present, and represents a positive integer, return it
+ * typecast to Number.
+ * Otherwise, parameter is invalid for page number, and return `null`.
+ * @param {string} queryPage `page` query parameter from URL
+ * @return {?number}
+ */
+export function pageFromQuery(queryPage) {
```

Comment:
Let's add some feature tests, something like:
```
Feature: Pagination

  Scenario: Being redirected

    When I open `/search?query=&page=-1`
    Then I should be on `/search?query=&page=1`

    When I open `/search?query=&page=0`
    Then I should be on `/search?query=&page=1`

    When I open `/search?query=&page=one`
    Then I should be on `/search?query=&page=1`

    When I open `/search?query=&page=last`
    Then I should be on `/search?query=&page=1`

    When I open `/search?query=&page=2.5`
    Then I should be on `/search?query=&page=1`
```

When I ran that locally initially it caused errors with the 'I should be on xyz' step again. Same as before the step which checks the url seems to run outside of the feature and interferes with the next step.
I was able to fix it by making that step await it's completion.
```
defineStep(/^I should be on.*? `(.*)`$/, async(pageName) => {
  let expectedUrl;
  if (pageName.startsWith('/')) {
    expectedUrl = `${url}${pageName}`;
  } else {
    expectedUrl = pages[pageName];
  }
  // TODO: update if a less verbose syntax becomes available.
  // See https://github.com/nightwatchjs/nightwatch/issues/861
  await client.url(currentUrl => {
    client.expect(currentUrl.value).to.eq(expectedUrl);
  });
});

```